### ISSUE TITLE
nothing happening after double clicking application

### ISSUE THREAD

**Issue by thewokgamur** [olegos2/mobox#230]
i tried to open feeding frenzy 2 but there is nothing happend but when i open the uninstall it's work perfectly fine how to fix this, wow64 version, turnip v5, dxvk 1-10.3

**Comment by bufan1015** [comment]
Yes, this is true for wine 8.1 and above, many applications only appear in the task manager
